### PR TITLE
[Cloud Security] Update vulnerability management integration title and docs

### PR DESCRIPTION
**Pull request by kfirpeled**
## What does this PR do?

This PR adds documentation for vulnerability management

## Screenshots

https://user-images.githubusercontent.com/61654899/231285341-cdedbd9c-6d11-4ac3-bea9-9e3f5e25b0db.mov

### CODE REVIEW

**Comment by kfirpeled** [comment]
@nick-alayil I have converted your initial draft for the documentation
Please fill in the missing pieces
Besides that, we should add a paragraph to the main doc as well, please fill that as well

File: packages/cloud_security_posture/docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Security Posture Management
 
-Use the Security Posture Management integration to maintain the **confidentiality, integrity, and availability** of your data in the cloud by continuously identifying configuration risks in your cloud infrastructure, like publicly exposed storage buckets and overly permissive networking objects. Read below to learn more about the components that make up security posture management; CSPM & KSPM. 
+Use the Security Posture Management integration to maintain the **confidentiality, integrity, and availability** of your data in the cloud by continuously identifying configuration risks and vulnerabilies in your cloud infrastructure, like publicly exposed storage buckets, overly permissive networking objects and Log4j vulnerability in workloads. Read below to learn more about the components that make up security posture management; CSPM, KSPM and CNVM.
```
Comment (Omolola-Akinleye) [inline packages/cloud_security_posture/docs/README.md:3]: Nit: `vulnerabilies` here is misspelled. `vulnerabilities`

Comment (nick-alayil) [reply packages/cloud_security_posture/docs/README.md]: Fixed typo. Thanks @Omolola-Akinleye 👍

**Review by Omolola-Akinleye — APPROVED** [review]

**Pull request merged**